src/app/components/tag/tag.component.ts

```diff
@@ -24,6 +24,8 @@ import { ArlasCollaborativesearchService, ArlasConfigService } from '../../servi
 import { ArlasTagService } from '../../services/tag/tag.service';
 import { ConfirmModalComponent } from '../confirm-modal/confirm-modal.component';
 import { ArlasBookmarkService } from '../../services/bookmark/bookmark.service';
+import { Aggregation, AggregationsRequest, AggregationResponse } from 'arlas-api';
+import { from } from 'rxjs';
```

Comment:
use import { from } from 'rxjs/observable/from';

Reply:
It's the old way to import, since rxjs 6 we have to import from `rxjs`

Reply:
See here : https://www.learnrxjs.io/operators/creation/from.html